firecrown/ccl/sources/sources.py

```diff
@@ -425,12 +424,7 @@ def read(self, sacc_data):
             The data in the sacc format.
         """
         tracer = sacc_data.get_tracer(self.sacc_tracer)
-        ell = getattr(tracer, 'ell').copy().flatten()
```

Comment:
Ohhh wait. @damonge would this be the `ell` values for the beam?

Reply:
This seems to be outdated. In any case, indeed beams come with an ell array attached to them. Although as I said below it's unlikely that CMB lensing maps will come with a beam (doesn't hurt to be general though).